content/docs/1.7.0/nodes-and-volumes/nodes/node-conditions.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Node Conditions
+weight: 7
+---
+
+Node conditions are introduced from Longhorn version 1.7.0 (`Multipathd`, `RequiredPackages` and `NFSClientInstalled`). They are used to check the environment settings of worker nodes to identify potential issues before any system impact.
```

Comment:
No need to mention any version and specific conditions in the paragraph. It is simply describing the purpose of node conditions. For example: https://kubernetes.io/docs/reference/node/node-status/#condition